2015-07-16 11:00:28,611 [6] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 147
2015-07-16 11:02:01,873 [15] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 147
2015-07-16 11:02:42,928 [28] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 147
2015-07-16 11:02:52,789 [28] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 147
2015-07-16 11:11:01,018 [40] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 147
2015-07-16 14:17:40,308 [41] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:20:49,686 [28] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:24:20,995 [25] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:31:54,601 [25] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:33:39,669 [17] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:33:39,674 [16] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:35:14,768 [5] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:38:11,907 [17] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:38:11,951 [11] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - An exception has been raised that is likely due to a transient failure. If you are connecting to a SQL Azure database consider using SqlAzureExecutionStrategy.
System.Data.Entity.Core.EntityException: An exception has been raised that is likely due to a transient failure. If you are connecting to a SQL Azure database consider using SqlAzureExecutionStrategy. ---> System.Data.Entity.Core.EntityException: The underlying provider failed on Open. ---> System.Data.Entity.Core.EntityException: An exception has been raised that is likely due to a transient failure. If you are connecting to a SQL Azure database consider using SqlAzureExecutionStrategy. ---> System.Data.SqlClient.SqlException: Клиенту не удалось установить соединение из-за ошибки, возникшей во время инициализации соединения до входа в систему. Возможны следующие причины ошибки: клиент попытался соединиться с неподдерживаемой версией SQL Server; сервер был слишком загружен и не мог принимать новые соединения; ресурсы сервера были ограничены (недостаточно памяти или превышено допустимое число соединений). (provider: Named Pipes Provider, error: 0 - С обоих концов канала отсутствуют процессы.) ---> System.ComponentModel.Win32Exception: С обоих концов канала отсутствуют процессы
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   в System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   в System.Data.SqlClient.TdsParserStateObject.SNIWritePacket(SNIHandle handle, SNIPacket packet, UInt32& sniError, Boolean canAccumulate, Boolean callerHasConnectionLock)
   в System.Data.SqlClient.TdsParserStateObject.WriteSni(Boolean canAccumulate)
   в System.Data.SqlClient.TdsParserStateObject.WritePacket(Byte flushMode, Boolean canAccumulate)
   в System.Data.SqlClient.TdsParser.SendPreLoginHandshake(Byte[] instanceName, Boolean encrypt)
   в System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean withFailover)
   в System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   в System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   в System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   в System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   в TaskApplication.Services.Concrete.IssueService.GetAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 133
2015-07-16 14:43:20,648 [32] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
2015-07-16 14:44:45,470 [33] ERROR TaskApplication.Services.Concrete.IssueService [(null)] - The object cannot be deleted because it was not found in the ObjectStateManager.
System.InvalidOperationException: The object cannot be deleted because it was not found in the ObjectStateManager.
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity, EntitySet expectedEntitySet)
   в System.Data.Entity.Core.Objects.ObjectContext.DeleteObject(Object entity)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Remove(Object entity)
   в System.Data.Entity.DbSet`1.Remove(TEntity entity)
   в TaskApplication.DataAccess.Repositories.GenericRepository`2.Delete(T entity) в g:\projects\TaskApplication\TaskApplication.DataAccess\Repositories\Concrete\GenericRepository.cs:строка 59
   в TaskApplication.Services.Concrete.IssueService.DeleteAllResolved() в g:\projects\TaskApplication\TaskApplication.Services\Concrete\IssueService.cs:строка 151
